pegasus/sites.v3/code.org/public/administrators.md

```diff
@@ -41,7 +41,7 @@ Code.org has partnered with more than <a href="/educate/district/partners" style
 
 ## Get started with curriculum and professional learning
 
-<%= view :course_wide_block, cta_link: CDO.studio_url('/pd/regional_partner_contact/new'), cta_text: 'Contact a Regional Partner', img: CDO.code_org_url('/shared/images/banners/small-teal-icons.png'), title: 'Grades K-12', description: "Code.org's professional development workshops support teachers with no background in computer science. We offer a high quality K-12 curriculum that can be used for a semester or full year. It includes daily lesson plans, detailed instructions, activity guides, assessments, and standards mappings, all free to use. In the U.S., Code.org's network of Regional Partners prepare teachers to successfully lead computer science classrooms." %>
+{{ administrators/course_wide_block }}
```

Comment:
How would you pass in the `cta_text`, `img`, `title`, etc params into the partial?

Reply:
I'm thinking for something like _this_ with so much content, it _should_ live entirely in the partial even in a world where you can pass arguments into partials.

Arguments into partials I see being useful for things like `CDO.studio_url` or `course_explorer_table` in this template

Reply:
Note that in this specific example, a new partial `views/administrators/course_wide_block.haml` has been created that contains all these values. This PR isn't just an abstract POC, it actually fully works